eng/psakefile.ps1

```diff
@@ -68,6 +68,13 @@ Task parser -Depends init {
     Build-PyQIR($pyqir.parser.name)
 }
 
+Task pyqir-tests -Depends init {
+    Invoke-LoggedCommand -workingDirectory (Join-Path $repo.root pyqir-tests) {
+        & $python -m pip install tox
```

Comment:
It looks like this runs in whatever virtualenv was active when `psakefile.ps1` was run. It may be worth creating a new virtualenv before installing `tox` to make sure to avoid making global changes to local dev environments.

Reply:
@cgranade The pyqir-tests folder defines a tox env which they are installed into.

Reply:
Right, understood, my question was about tox itself as it looks like that task doesn't activate an environment before installing tox.

Reply:
This problem existed before this PR, so I created #55 to follow up on it.